libsel4bench/arch_include/arm/cpu/cortex-a35/sel4bench/cpu/events.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2017, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#pragma once
+#define SEL4BENCH_EVENT_BUS_ACCESS_LD         0x60
```

Comment:
It would be good to add a comment with the source of this. See e.g. what https://github.com/seL4/seL4_libs/blob/master/libsel4bench/arch_include/arm/cpu/cortex-a72/sel4bench/cpu/events.h says. Seems this may end it in lot of redundant comment, but this may help when we try to unify things one day.

Reply:
A72 is clearly the outlier; none of the other `events.h` files have similar.

Doesn't mean it is a bad idea, but I think this is no worse than existing implementations overall. I don't have time to go back and find the original reference at this point, but more than happy if someone wants to do that comprehensively for all `events.h` headers.

Reply:
I do believe you that you have looked them up and they work. Could you at least add a short note here that these values come from the TRM and are aligned with the ARM ARM, so we have something here that give a clue where to look. It's a bit better than just having magic values flying around where it's unclear where they come from.